tasks/systemd.yml

```diff
@@ -1,4 +1,13 @@
 ---
+
+- name: Systemd - Get packages facts
+  package_facts:
+    manager: "auto"
+
+- name: Systemd - Set haproxy version
+  set_fact:
+    haproxy_major_version: "{{ ansible_facts.packages.haproxy[0].version.split('.')[0] }}.{{ ansible_facts.packages.haproxy[0].version.split('.')[1] }}"
```

Comment:
Why do you bother with `split()` ? The ansible's [version](https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison) test is capable of dealing with long version numbers.

Reply:
I thought that ansible's version was not able to do that ! Fixed ;)